### PR TITLE
Add conformance test suite info

### LINKED ISSUE #653: Conformance test suites and the OSBAPI Working Group
One of the [goals](https://www.openservicebrokerapi.org/) of the OSBAPI group has been:
> Create a conformance test suite to verify both consumer and producer behaviours

Whilst the group adopted the [osb-checker](https://github.com/openservicebrokerapi/osb-checker) from Microsoft in August 2018, the group has found it hard to maintain this test suite (which is focussed on verifying service broker compliance with the spec). As a result, the test suite does not support the latest version of the specification, and we have seen other test suites emerge in the community.

When adding new features to the spec, there is already a considerable amount of work that has to be done by the community, including validating implementations, updating the spec and updating multiple swagger/openapi documents. Updating a test suite which, for many of us may be written in a programming language we are unfamiliar with, as part of every change to the specification seems to be unfeasible.

@fmui and I have been talking about this at length, and we'd like to propose a couple of changes:

1. We remove the goal from the [https://www.openservicebrokerapi.org](https://www.openservicebrokerapi.org) website about creating a conformance test suite.
2. We enhance the project's [getting started](gettingStarted.md) page with a list of test suites which service broker authors could use.

The test suites that support the latest version of the spec and are being maintained today include:
1. [osb-checker-kotlin](https://github.com/evoila/osb-checker-kotlin) by the team at Evoila including @chbrinker
2. [osb-checker](https://github.com/openservicebrokerapi/osb-checker), which was originally donated by @arschles at Microsoft but has subsequently had a lot of work put into a redesign by @leonwanghui 

There may be more test suites out there, and like the various libraries we link to, we could simply link to these test suites for folks to choose from. This list can easily be extended through pull requests to the getting started page in this repository.

Before proceeding with this, we'd like to get your feedback and will be discussing this on the next OSBAPI call on the 9th April.

### PR DESCRIPTION
**Pull request by mattmcneeney**
Closes #653 

@leonwanghui @chbrinker What do you think about this? Are the list of maintainers correct?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,3 +46,26 @@ wanting to implement a Platform or Service Broker.
...
 releases, and which platforms are currently supporting them.
+
+## Testing conformance to the Open Service Broker API
+
+One of the [goals](https://www.openservicebrokerapi.org/#goals) of the Open
+Service Broker API project is to **provide conformance test suites to verify
+compliance with the specification**.
+
+The conformance test suites below can be used to verify Service Broker behaviour
+against different versions of the specification:
```
Comment (mattmcneeney) [inline README.md]: Add a comment about PR'ing new ones in?

Comment (chbrinker) [reply README.md]: Maybe depends on the ch checkers project specific contribution guidelines? So should be done on each projects start page? Maybe add this convention here?

Comment (mattmcneeney) [reply README.md]: I'm not sure I understand @chbrinker - do you mean which versions?

Comment (chbrinker) [reply README.md]: Was a misunderstanding. Did not read your question correctly. I was thinking you meant contributions to the checker frameworks. But now I think you meant contributing new frameworks to the list of checkers. Makes more sense. You should add this info.

**Comment by leonwanghui** [comment]
I think this PR is cool to be merged 👍

**Pull request merged**